PointOfDispensingApp/src/components/PatientLookupPage.vue

```diff
@@ -180,13 +221,18 @@
     },
     data () {
 	return {
+    date: new Date().toISOString().substr(0, 10),
+    dateFormatted: this.formatDate(new Date().toISOString().substr(0, 10)),
 		selectedId: -1, //initializing the ID of the selected row to be something that is obviously invalid
     selectedFamilyName: '',
     selectedGivenName: '',
     selectedDOB: '',
     selectedLineAddress: '',
     selectedCity: '',
     selectedPostalCode: '',
+    birthdateRules: [
+        (v) => v.length == 10 || "DOB must be in format MM-DD-YYYY"
```

Comment:
```suggestion
        (v) => v.length == 10 || "DOB must be in format MM/DD/YYYY"
```
These rules dont guard against 99/99/9999 for example. There's a birthdate rules list from the Patient Registration app that you could build off of.

Reply:
I was thinking of leaving that as is, since the use case here will not be submitting info, rather it is retrieving info.

Trying to look for incorrect values would be ok.

Reply:
I agree